app/src/main/java/com/example/kuba10/mypokemonplaces/AddPlaceFragment/AddPlaceFragment.java

```diff
@@ -1,169 +0,0 @@
-package com.example.kuba10.mypokemonplaces.AddPlaceFragment;
-
-
-import android.content.Context;
-import android.os.Bundle;
-import android.support.v4.app.Fragment;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.Button;
-import android.widget.ImageView;
-import android.widget.TextView;
-
-import com.example.kuba10.mypokemonplaces.ChooseFragment.ChooseFragment;
-import com.example.kuba10.mypokemonplaces.Constants;
-import com.example.kuba10.mypokemonplaces.FragmentListener;
-import com.example.kuba10.mypokemonplaces.Model.PokePlace;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.R;
-import com.google.android.gms.maps.model.LatLng;
-import com.squareup.picasso.Picasso;
-
-import java.util.ArrayList;
-import java.util.Date;
-
-import butterknife.BindView;
-import butterknife.ButterKnife;
-
-
-
-public class AddPlaceFragment extends Fragment {
-
-    private FragmentListener fragmentListener;
-    private ArrayList<PokemonGo> pokemonGo_data_list;
-    private PokemonGo selectedPokemon;
```

Comment:
Ustaw zmienne w kolejności alfabetycznej

Reply:
ustawia sie je po klasie obiektu czy po nazwie zmiennej?

Reply:
Część ustawia po nazwie zmiennej (czyli w sumie tak jak masz). Ja zawsze ustawiam po typie, dlatego od razu dałem komentarz. Jak Ci tak pasuje to zostaw (tylko nazwę listy zmień, plis :smile: )